.editorconfig

```diff
@@ -0,0 +1,5 @@
+[*]
+insert_final_newline = true
+indent_size = 4
+indent_style = tab
+tab_width = 4
```

Comment:
You could add `trim_trailing_whitespace = true` to trim useless trailing whitespaces.

Reply:
done